html/rendering/non-replaced-elements/the-fieldset-element-0/legend-float.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<title>legend and float</title>
+<link rel=match href=legend-float-ref.html>
+<style>
+  fieldset { margin: 0; padding: 0; border: none; width: 100px; height: 50px; background: red; }
+  legend { width: 100px; height: 50px; background: lime; padding: 0; }
+  .left { float: left; }
+  .right { float: right; }
+</style>
+<p>There should be no red.</p>
+<fieldset><legend class=left></legend></fieldset>
```

Comment:
Not sure how you expect this to fail in an imaginary faulty browser (all pass this, right?), since you're removing the potential effects a rendered legend could have (no border or padding on the fieldset).

Reply:
The faulty browser would be Chrome M55 per https://bugs.chromium.org/p/chromium/issues/detail?id=675676 ("Legend float in fieldset makes legend disappear").

Reply:
Oh, I see. I misguessed what you were trying to test. I thought you wanted to test that a floated legend isn't accepted as a rendered legend. But maybe you *should* write the test so that it also renders differently depending on whether it becomes a rendered legend or not.

Reply:
There's a separate test for that at https://github.com/web-platform-tests/wpt/pull/12567

Reply:
Ok, fine then.